docs/syntax.md

```diff
@@ -21,6 +25,12 @@ Below you will find the step syntax next to the name of the method it utilizes.
 - `<GK> [the] (deployment|hpa|horizontalpodautoscaler|service|pdb|poddisruptionbudget|sa|serviceaccount) <any-characters-except-(")> is in namespace <any-characters-except-(")>` kdt.KubeContext.ResourceInNamespace
 - `<GK> [I] scale [the] deployment <any-characters-except-(")> in namespace <any-characters-except-(")> to <digits>` kdt.KubeContext.ScaleDeployment
 - `<GK> [the] (clusterrole|clusterrolebinding) with name <any-characters-except-(")> should be found` kdt.KubeContext.ClusterRbacIsFound
+- `<GK> [I] (add|remove) ?<any-characters-except-(")> as trusted entity to iam role <any-characters-except-(")>` kdt.AwsContext.IamRoleTrust
+- `<GK> [I] (add|remove) cluster shared iam role` kdt.AwsContext.ClusterSharedIamOperation
```

Comment:
I had moved this 2 steps under the "AWS Steps" tittle, why do we need them up here @Kyle-Wong?

Reply:
That might have been a mistake when I merged into my PR branch. The steps don’t need to be up here